.github/workflows/ci.yml

```diff
@@ -490,57 +553,31 @@ jobs:
         run: |
           minikube profile list
           minikube status
-      - name: Install stable Fluvio CLI and start cluster
+
+      - name: Setup for upgrade test
         run: |
+          gh auth login --with-token < <(echo ${{ secrets.GITHUB_TOKEN }})
           curl -fsS https://packages.fluvio.io/v1/install.sh | bash
-          ~/.fluvio/bin/fluvio cluster start
-          ~/.fluvio/bin/fluvio version
-      # Build, upgrade and test current version
-      - name: Build
-        run: |
-          make RELEASE=release TARGET=x86_64-unknown-linux-musl build_test
-      - name: Setup installation pre-requisites
-        run: |
-          make RELEASE=true TARGET=x86_64-unknown-linux-musl  k8-setup
-      - name: Make image
-        run: make RELEASE=true minikube_image
-      - name: Upgrade cluster
-        run: |
-          cargo run --release --bin fluvio -- cluster upgrade --chart-version=$(cat VERSION) --develop 
-          cargo run --release --bin fluvio -- version 
-      - name: Run smoke-test-k8-tls-root
-        timeout-minutes: 5
-        run: |
-          make RELEASE=true TARGET=x86_64-unknown-linux-musl UNINSTALL=noclean smoke-test-k8-tls-root
-          cargo run --release --bin flv-test --	smoke --develop --disable-install -- --producer-iteration=1000
+
+      - name: Start sccache server
+        run: sccache --start-server
+
+      - name: Run upgrade test
+        env:
+          TEST_DATA_BYTES: 10000
+        uses: nick-invision/retry@v2
+        with:
+          timeout_minutes: 45
+          max_attempts: 3
+          command: |
+            export PATH=~/.fluvio/bin:$PATH
+            cd tests
+            ./upgrade-test.sh `gh release -R infinyon/fluvio list | head -3 | tail -2 | awk '{print $1}' | sed 's/v//' | tac | xargs`
```

Comment:
This can probably be simplified to

```
gh api repos/infinyon/fluvio/releases -q '.[2,1].tag_name' | sed 's/v//'
0.8.1
0.8.2
```

Reply:
Also instead of using backticks for inline commands it is better practice to use `$(...)`